.env.example

```diff
@@ -10,7 +10,7 @@ CSMM_LOG_CHECK_INTERVAL=3000
 CSMM_LOG_COUNT=50
 
 # Comma separated list of steam IDs for users that get extended control
-CSMM_ADMINS=76561198028175941,76561198028175941
+CSMM_ADMINS=
```

Comment:
I think it'd be better to add some placeholder IDs here or something. The purpose of these IDs is ~so I have a backdoor into every instance muahahah~ to show people how to pass multiple IDs in. In the past, "Comma separated list of steam IDs" wasn't clear enough for people.

So I'd say either add placeholder IDs to show this 
OR
Extend the comment with an example

Reply:
Haha I added another commit.  Commented out that line so users have to choose to create a backdoor :) 

How's that sound?